server/test/integration/com/thoughtworks/go/server/service/ValueStreamMapServiceIntegrationTest.java

```diff
@@ -73,6 +74,11 @@
     private HttpLocalizedOperationResult result;
     private Username username;
 
+    //  Hack to ignore MDU requests from MaterialUpdateService, as tests update the DB directly.
```

Comment:
Might be useful to have this comment in the other places, too. :) Or maybe move it to one single place (I know it's hard to find such a place).